spec/controllers/contact_controller_spec.rb

```diff
@@ -48,8 +50,8 @@
   end
   context 'with contact e-mail set' do
     before do
-      AppConfig.first
-        .update_attributes(contact_link_location: 'contact@example.com')
+      allow(@ac).to(
+        receive(:contact_link_location).and_return('contact@example.com'))
```

Comment:
FYI, rubocop is going to start complaining about these trailing parentheses (see [here](https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/style/closing_parenthesis_indentation.rb)); if we need to split these stubs into multiple lines how about something like:

```
allow(@ac).to receive(:contact_link_location)
  .and_return('contact@example.com')
```

This should pass rubocop and avoids having to wrap the stub statement in parentheses.